blaze/json_dumps_trusted.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function
+
+import types
+from collections import Callable
+
+from .dispatch import dispatch
+from functools import partial, wraps
+
+from .json_dumps import json_dumps
+
+
+json_dumps_trusted_ns = dict()
```

Comment:
Using `json_dumps.funcs.copy()` still retains the `json_dumps` function name, which ultimately triggers an exception in multipledispatch's inner dispatching logic; I have to do `json_dumps_trusted.register(types)(func)` to ensure the `json_dumps_trusted` function name is updated.